### PR TITLE
Vim, VimBox, CSS

### PR DESCRIPTION
**Pull request by Robert-M-Muench**
Now with prior rebase.

### CODE REVIEW

File: cheatsheets/Vim.rb [hunk 1]
```diff
@@ -209,6 +209,37 @@
             command '{Visual}~'
             name 'Switch case of highlighted text'
         end
+        entry do
+            command '{Visual}U'
+            name 'Uppercase highlighted text'
+    end
+        entry do
+              command 'SHIFT+I<comment-char>ESC'
+              name 'Comment Vertical Selection'
```
Comment (Kapeli) [inline cheatsheets/Vim.rb]: Please follow the [Guidelines](https://github.com/Kapeli/cheatsheets#guidelines). No title case in entry names.

File: cheatsheets/Vim.rb [hunk 2]
```diff
@@ -209,6 +209,37 @@
...
+    category do
+        id 'Information'
+
+        entry do
+          command '%'
+          name 'Jump to matching brace'
+        end
+        entry do
+          command '*'
+          name 'Highlight all orrurences of word under cursor'
```
Comment (Kapeli) [inline cheatsheets/Vim.rb]: Typo: occurrences instead of orrurences.

**Comment by Kapeli** [comment]
I'm going to merge and fix the issues.

Regarding the CSS cheat sheet: I'll accept it but won't make it show within Dash yet. It's too barebones right now. Please improve it in the future.

**Comment by Robert-M-Muench** [comment]
Thanks. And yes the CSS is just a beginning with those parts I don't remember if not used every day ;-) will grow.

**Pull request merged**